common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1073,6 +1073,16 @@
          */
         public static final String BROWSER_EXT_INSTALL_PREFIX = "msauth://";
 
+        /**
+         * Prefix in the redirect from WebCP.
```

Comment:
This scheme we're using, does this provide a deep link into CompanyPortal?

Reply:
In other words, can we say what it does in a comment or a link to a doc?

Reply:
We steered away from deep linking because any apps can 'declare' the ownership of the link (in such case, Android will let the user choose which app to redirect to - and we don't even want this to happen).

Therefore, what we do here is to 'look for the pattern' here and launch an intent directly with the app package name. we won't be using the URI directly.

Reply:
Again, could we add a comment saying what we're doing and why?